docs/guides/oracle-operator-manual.md

```diff
@@ -6,20 +6,20 @@ The daemon also fetches historical stETH token price (shifted by fifteen blocks)
 
 ## TL;DR
 
-1. Generate an Ethereum address and propose it as an oracle address via the "Add Member" button in the app UI: [Mainnet] / [Görli].
-2. Facilitate the DAO members to approve your oracle address.
-3. Launch and sync an Ethereum 1.0 node with JSON-RPC endpoint enabled.
-4. Launch and sync a Beacon Chain node with API endpoint enabled. (one of Lighthouse, Prysm, Teku)
-5. Launch the oracle daemon as a docker container.
-
-[mainnet]: https://mainnet.lido.fi/#/lido-dao/0x442af784a788a5bd6f42a01ebe9f287a871243fb/
-[görli]: https://testnet.testnet.fi/#/lido-testnet-prater/0xbc0b67b4553f4cf52a913de9a6ed0057e2e758db/
+1. Generate an Ethereum address.
+2. Share your address with public in case of mainnet it should be twitter and the research forum, in case of testnet it may be telegram or discord.
+3. Ask DAO members to prepare a vote to add addresses to the oracle set.
```

Comment:
I propose to remove point 3. 
And to move point 2 before 6. And write it like this:
2. In case of mainnet share your address and intention to join the Oracle set with public. You need to publish it on twitter and also write a message with a twitter link under Onboarding post on [the Research forum](https://research.lido.fi/) .